Sprint-1/exercises/count.js

```diff
@@ -1,6 +1,6 @@
 let count = 0;
 
-count = count + 1;
+count = count + 1; //count increased by one so addion operaiton is perfomed here.
```

Comment:
That's the right idea. Can you get more specific about what "=" is doing in this line?

Reply:
I make some modifications. thank you.